plasmapy/tests/helpers/__init__.py

```diff
@@ -0,0 +1,32 @@
+"""Test helper functionality for PlasmaPy and affiliated packages."""
+
+# This file contains several commented out import statements.  These
+# statements will be uncommented out over the course of several pull
+# requests that were each originally part of #728.  The blank lines
+# between the import statements will hopefully simplify automatic merging.
+
+# from plasmapy.tests.helpers.actual import ActualTestOutcome
+
+# from plasmapy.tests.helpers.cases import AttrTestCase, FunctionTestCase, MethodTestCase
```

Comment:
I think this is fine for your working branch, but should be removed for the PR.  It refers to code that does not exist yet and that shouldn't happen in the `master` branch.

Reply:
The reason why I have this in here is because I have a whole bunch of simultaneous closely related pull requests that are each going to need to make changes to this file, and I'm hoping that this will prevent the need to resolve merge conflicts across ~7 PRs and allow the PRs to be cleanly separated from each other.  Pretty much...if you let me bend the rules a little bit just this once...it'll make my life a little easier!

Reply:
Fine, but it will cost you a beer at the next in-person APS. 😁

Reply:
> Fine, but it will cost you a beer at the next in-person APS. grin

Thanks!  I'll try to remember that for 2026.